app/scripts/controllers/permissions/selectors.test.js

```diff
@@ -113,4 +119,89 @@ describe('PermissionController selectors', () => {
       expect(selected2).toBe(getPermittedAccountsByOrigin(state2));
     });
   });
+
+  describe('getPermittedChainsByOrigin', () => {
+    it('memoizes and gets permitted chains by origin', () => {
```

Comment:
How is it memoizing them? I'm not seeing that in the selector?

Reply:
i just copied this from the other scenario, but 

https://github.com/reduxjs/reselect#:~:text=Reselect%20exports%20a%20createSelector%20API%2C%20which%20generates%20memoized%20selector%20functions